cmds/dump.c

```diff
@@ -1584,6 +1584,7 @@ int command_dump(int argc, char *argv[], struct opts *opts)
 			},
 		};
 
+		disable_argspec_color();
```

Comment:
If you call it in `setup_color()`, I guess this change is not needed since we turn off color when redirecting to a file.